### PR TITLE
[v0.6.8] Introduce broadcast tx commit timeout

### PR DESCRIPTION
**Pull request by charithabandi**

### CODE REVIEW

File: cmd/kwild/config/flags.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ func AddConfigFlags(flagSet *pflag.FlagSet, cfg *KwildConfig) {
 
 	// Chain RPC flags
 	flagSet.StringVar(&cfg.ChainCfg.RPC.ListenAddress, "chain.rpc.listen-addr", cfg.ChainCfg.RPC.ListenAddress, "Chain RPC listen address")
+	flagSet.Var(&cfg.ChainCfg.RPC.BroadcastTxTimeout, "chain.rpc.broadcast-tx-timeout", "Chain RPC broadcast transaction timeout")
```
Comment (jchappelow) [inline cmd/kwild/config/flags.go:47]: We've been using `Var` for the other durations, so no objection, but I wonder why we don't use `DurationVar`.  Seems to exist.  Maybe just doesn't make help anything.

**Comment by jchappelow** [comment]
Will you make this PR for `main` as well?

**Comment by charithabandi** [comment]
> Will you make this PR for `main` as well?

Yup, done. https://github.com/kwilteam/kwil-db/pull/684

File: cmd/kwild/config/flags.go (hunk 1 above)
Comment (charithabandi) [reply cmd/kwild/config/flags.go:47]: not sure why we switched to Var, I am sure we used to use DurationVar before.

**Review by jchappelow — APPROVED** [review]

**Pull request merged**